src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -39,9 +39,9 @@
             </div>
         </esfaFeatureEnabled>
     </div>
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-one-third">
-            <form method="get">
+    <form method="get">
```

Comment:
In Provider we also have `autocomplete="off"` attribute on the form.

Reply:
👍